csharp/src/SeedLang/Runtime/NativeFunction.cs

```diff
@@ -40,16 +43,77 @@ internal class NativeFunction : IFunction {
 
   // The static class to define all the build-in native functions.
   internal static class NativeFunctions {
+    public const string HasNext = "hasnext";
```

Comment:
What is `hasnext` for? Python doesn't provide this method. Is it for determining the end of an iterator without an exception mechanism?

Reply:
`hasn't` build-in function is removed as well, because of removal of `iter`.